tools/publish_release.py

```diff
@@ -61,7 +61,7 @@ def release_channel():
   return channel
 
 def publish_release(github, release_id):
-  data = dict(draft=False)
+  data = dict(draft=False, prerelease=True)
```

Comment:
oh crap- sorry this must have got added when I did a `git commit -a`; backing this out...